src/css/mediaelementplayer.css

```diff
@@ -383,28 +389,20 @@ Reference: http://blog.rrwd.nl/2015/04/04/the-screen-reader-text-class-why-and-h
 	left: 18px;
 }
 
-/*
-.mejs-controls .mejs-time-rail:hover .mejs-time-handle {
-	visibility:visible;
-}
-*/
 /* End: Progress Bar */
 
 /* Start: Fullscreen */
-.mejs-controls .mejs-fullscreen-button button {
+.mejs-fullscreen-button > button {
 	background-position: -32px 0;
 }
 
-.mejs-controls .mejs-unfullscreen button {
+.mejs-unfullscreen button {
```

Comment:
Yes, good call. Please add.

Also, separately, could you hook up the `<track>` tag in the latest alpha html test page? Would be great to always confirm captions are working in the full player. Since the SRT is probably very lightweight this should maybe just be part of the repo, for testing?